hardhat.config.ts

```diff
@@ -1,143 +1,158 @@
-task("verify-contracts", "Verify solidity source", verifyContracts);
+import { task } from "hardhat/config";
+import "@nomiclabs/hardhat-waffle";
+import "hardhat-contract-sizer";
+import "hardhat-log-remover";
+import "hardhat-deploy";
+import "@nomiclabs/hardhat-ethers";
+import "@nomiclabs/hardhat-etherscan";
+import "solidity-coverage";
+import "hardhat-gas-reporter";
+import exportDeployments from "./scripts/tasks/exportDeployments";
+import verifyContracts from "./scripts/tasks/verifyContracts";
+import { BLOCK_NUMBER } from "./constants/constants";
+import { TEST_URI } from "./scripts/helpers/getDefaultEthersProvider";
+
+require("dotenv").config();
+
+process.env.TEST_MNEMONIC =
+  "test test test test test test test test test test test junk";
+
+// Defaults to CHAINID=1 so things will run with mainnet fork if not specified
+const CHAINID = process.env.CHAINID ? Number(process.env.CHAINID) : 1;
+
+export default {
+  accounts: {
+    mnemonic: process.env.TEST_MNEMONIC,
+  },
+  paths: {
+    deploy: "scripts/deploy",
+    deployments: "deployments",
+  },
+  solidity: {
+    version: "0.8.4",
+    settings: {
+      optimizer: {
+        runs: 200,
+        enabled: true,
+      },
+    },
+  },
+  networks: {
+    hardhat: {
+      accounts: {
+        mnemonic: process.env.TEST_MNEMONIC,
+      },
+      chainId: CHAINID,
+      forking: {
+        url: TEST_URI[CHAINID],
+        blockNumber: BLOCK_NUMBER[CHAINID],
+        gasLimit: 8e6,
+      },
+    },
+    mainnet: {
+      url: process.env.TEST_URI,
+      chainId: 1,
+      accounts: {
+        mnemonic: process.env.MAINNET_MNEMONIC,
+      },
+    },
+    kovan: {
+      url: process.env.KOVAN_URI,
+      chainId: 42,
+      accounts: {
+        mnemonic: process.env.KOVAN_MNEMONIC,
+      },
+    },
+    bsc_mainnet: {
+      url: process.env.BSC_MAINNET_URI,
+      chainId: 56,
+      accounts: {
+        mnemonic: process.env.BSC_MAINNET_MNEMONIC,
+      },
+    },
+    avax: {
+      url: process.env.AVAX_URI,
+      chainId: 43114,
+      accounts: {
+        mnemonic: process.env.AVAX_MNEMONIC,
+      },
+    },
+    fuji: {
+      url: process.env.FUJI_URI,
+      chainId: 43113,
+      accounts: {
+        mnemonic: process.env.FUJI_MNEMONIC,
+      },
+    },
+  },
+  namedAccounts: {
+    deployer: {
+      default: 0,
+      1: "0xF8368119Bb1073Cf01B841848725d81b542A4c19",
+      42: "0x422f7Bb366608723c8fe61Ac6D923023dCCBC3d7",
+      56: "0xF8368119Bb1073Cf01B841848725d81b542A4c19",
+      43114: "0x422f7Bb366608723c8fe61Ac6D923023dCCBC3d7",
+      43113: "0x422f7Bb366608723c8fe61Ac6D923023dCCBC3d7",
+      1313161554: "0x46B4E6143Fb6ded2e5FBd87887Ef4f50f716dcA0",
+      1313161555: "0x46B4E6143Fb6ded2e5FBd87887Ef4f50f716dcA0",
+    },
+    owner: {
+      default: 0,
+      1: "0x77DA011d5314D80BE59e939c2f7EC2F702E1DCC4",
+      42: "0x92Dd37fbc36cB7260F0d2BD09F9672525a028fB8",
+      56: "0xF8368119Bb1073Cf01B841848725d81b542A4c19",

```

Comment:
can we change this to 0x77?

Reply:
Addressed in https://github.com/ribbon-finance/ribbon-v2/pull/300/commits/e11a3cd85afed42c4bee272f44df8f9afe17bd31